schematic/models/validate_attribute.py

```diff
@@ -1621,33 +1683,33 @@ def _run_validation_across_targets_set(
                 column_names=column_names,
             )
 
-            # Do the validation on both columns
-            if "matchNone" in val_rule:
-                # Look for repeats between the source manifest and target_column, if there are
-                # repeats log the repeat value and manifest
-                repeat_values = manifest_col[manifest_col.isin(target_column)]
+            # Drop all missing values before checking sets
+            manifest_col.dropna(inplace=True)
+            target_column.dropna(inplace=True)
 
-                if repeat_values.any():
-                    repeat_manifest_log[target_manifest_id] = repeat_values
-            else:
-                # Determine elements in manifest column that are missing from the target column
-                missing_values = manifest_col[~manifest_col.isin(target_column)]
+            validation_output.target_manifests.append(target_manifest_id)
+
+            # Find values missing in checked column vs target column, and vice versa
+            missing_target_values = manifest_col[~manifest_col.isin(target_column)]
+            missing_manifest_values = target_column[~target_column.isin(manifest_col)]
+
+            if missing_target_values.empty and missing_manifest_values.empty:
+                validation_output.matching_manifests.append(target_manifest_id)
+
+            # Add missing values to SetValidationOutput
+            if not missing_target_values.empty:
+                validation_output.missing_target_values[
+                    target_manifest_id
+                ] = missing_target_values
```

Comment:
Is it possible that `validation_output.missing_target_values[target_manifest_id]` would already have data contained within it, or this to cause data to be overwritten?

Reply:
Nope! This method gets iterated over a list of manfiests. A method like `syn.GetChildren` gets the synapse ids, and these get downloaded and read in as `pandas.DataFrame`. So each `target_manifest_id` should only be called by this method once.